README.md

```diff
@@ -26,9 +26,51 @@ A library and CLI app for rendering project templates.
 1. To use as a CLI app: `pipx install copier`
 1. To use as a library: `pip install copier`
 
-## Quick usage
+## Quick start
 
--   Use it in your Python code:
+To create a template:
+
+```bash
+📁 my_copier_template ------------------------ # your template project
+├── 📄 copier.yml ---------------------------- # your template configuration
+├── 📁 .git ---------------------------------- # your template is a git repository
+├── 📁 {{project_name}} ---------------------- # a folder with a templated name
+│   └── 📄 {{module_name}}.py.jinja ---------- # a file with a templated name
+└── 📄 {{_copier_conf.answers_file}}.jinja --- # answers are written here
```

Comment:
I think this explains it better. I know it's a nitpick, sorry.

```suggestion
└── 📄 {{_copier_conf.answers_file}}.jinja --- # answers are recorded here
```

Reply:
No problem, thanks :smile: